README.md

```diff
@@ -18,7 +18,65 @@ with Bazel.
 
 ## Setup
 
-To be documented.
+### Using npm and Node.js rules
+
+We recommend you use npm to install the PostCSS build rules if you use the
+[Node.js build rules][rules_nodejs], and **strongly recommend** using npm
```

Comment:
does the 'Using repository rules' alternative below have any advantages? Why not give users a single install instruction?

Reply:
I believe users may prefer the standard repository rules approach rather than being required to set up rules_nodejs, despite the advantages available for installing via npm.

Reply:
well I think rules_nodejs is pretty minimal, just an http_archive and a npm_install rule and no dependencies, but maybe there's a case for this, you know your own users.

I would point out this increases the surface area you need to test - ideally you'd want bazel-in-bazel integration tests to ensure the install and repository rules work following your instructions.

Reply:
Good point - filed as #44.